examples/paho-mqtt/Makefile

```diff
@@ -0,0 +1,45 @@
+APPLICATION = paho-mqtt-example
+
+# If no BOARD is found in the environment, use this default:
+BOARD ?= native
+
+# This has to be the absolute path to the RIOT base directory:
+RIOTBASE ?= $(CURDIR)/../..
+
+# Comment this out to disable code in RIOT that does safety checking
+# which is not needed in a production environment but helps in the
+# development process:
+DEVELHELP ?= 1
+
+# Change this to 0 show compiler invocation lines by default:
+QUIET ?= 1
+
+#####################
+USEPKG += paho-mqtt
+USEMODULE += ipv6_addr
+USEMODULE += lwip_ipv6
+USEMODULE += lwip_ipv6_autoconfig
+USEMODULE += lwip lwip_sock_ip lwip_netdev
+USEMODULE += lwip_udp lwip_sock_udp
+USEMODULE += lwip_tcp lwip_sock_tcp
+USEMODULE += lwip_sock_async
+USEMODULE += sock_async_event
+USEMODULE += shell
+USEMODULE += shell_commands
+USEMODULE += ps
+USEMODULE += netdev_default
+
+ifeq ($(BOARD),native)
+  USEMODULE += lwip_ethernet
+endif
+
+LOG_LEVEL ?= LOG_INFO
```

Comment:
LOG_LEVEL is already set as LOG_INFO by default. Other than that I wouldn't use `LOG_*` function in the application code, just `printf`/`puts`.
For example, right now, by default, when subscribing to a topic, the received message is not printed: this gives the impression that it doesn't work.